.travis.yml

```diff
@@ -21,6 +21,9 @@ before_install:
   - cd coalahtml/_coalahtml
   - bower install
   - cd -
+  # colorama needs to be upgraded to fix error when installed colorama 
+  # version and required are different.
+  - sudo pip3 install -U colorama
```

Comment:
Sure, you can, but it'll need to be solved to make CI work IIUC. So it'll be nice if it go either with this PR or before it.